ci/run.sh

```diff
@@ -66,3 +64,32 @@ case ${TARGET} in
     *)
         ;;
 esac
+
+if [ "${NOLIBSTDBUILD}" = "1" ]; then
+    echo "Whether libstd builds with this stdsimd is not tested!"
+else
+    echo "Testing that libcore and libstd build with this stdsimd..."
+    stdsimd="$(pwd)"
+
+    case ${TARGET} in
+        *apple*)
+            export RUSTC_DIR=~/rustc
+            ;;
+        *windows*)
+            export RUSTC_DIR=~/rustc
+            ;;
+        *)
+            export RUSTC_DIR=/rustc
+            ;;
+    esac
+
+    git clone --depth 1 https://github.com/rust-lang/rust.git "${RUSTC_DIR}"
+    cd "${RUSTC_DIR}"
+    git submodule sync
+    ./x.py clean
+
+    cp -r "${stdsimd}"/* src/stdsimd/
```

Comment:
To let `git` do the sync'ing, I have to update the `.gitmodules` to the repo of the code, then I have to go into `src/stdsimd` and fetch the repo, pull in the right commit, and then commit the upgrade of stdsimd anyways in the rust-lang/rust repo. 

I thought it would just be easier to overwrite the `stdsimd` code in rust-lang/rust with the one that travis already checked out. I might need to commit the changes though.

Reply:
Why not just do something like this:
```
cd src/stdsimd
git remote add CI ../../path/to/stdsimd
git fetch CI
git checkout (commit)
```

Reply:
I'll give that a try. Doesn't one still need to update the `.gitmodules` file ?

Reply:
fatal: reference is not a tree: 129db7fd19883e88c1ae3514a09a8b49fb77ece0

Reply:
The code you suggested produced the exact same error.